django/contrib/auth/management/commands/createsuperuser.py

```diff
@@ -20,6 +20,7 @@ class NotRunningInTTYException(Exception):
 class Command(BaseCommand):
     help = 'Used to create a superuser.'
     requires_migrations_checks = True
+    private_options = ('username', )
```

Comment:
I've fixed this in the latest revision. The failing test needed a small adjustment.